Framework/DataHandling/src/SaveTBL.cpp

```diff
@@ -83,16 +83,28 @@ void SaveTBL::exec() {
         std::string groupHeading = columnHeadings[columnIndex];
         if (ws->getColumn(groupHeading)->type() != "int") {
           file.close();
-          remove(filename.c_str());
-          throw std::runtime_error(groupHeading +
-                                   " Column must be of type \"int\"");
+          int error = remove(filename.c_str());
+          if (error == 0)
```

Comment:
Is the error code check correct here? http://www.cplusplus.com/reference/cstdio/remove/ suggests that `0` marks success yet we are throwing an error on that scenario?